lib/Factory/ModuleTemplateFactory.php

```diff
@@ -168,6 +168,7 @@ private function createFromXml(\DOMElement $xml): ModuleTemplate
         $template->dataType = $this->getFirstValueOrDefaultFromXmlNode($xml, 'dataType');
         $template->title = $this->getFirstValueOrDefaultFromXmlNode($xml, 'title');
         $template->thumbnail = $this->getFirstValueOrDefaultFromXmlNode($xml, 'thumbnail');
+        $template->isVisible = $this->getFirstValueOrDefaultFromXmlNode($xml, 'isVisible');
```

Comment:
This will return a string, so we need to test it with `===` to convert to a boolean.